SyntheticsRunTestsTask/utils.ts

```diff
@@ -1,3 +1,20 @@
+import * as task from 'azure-pipelines-task-lib/task'
+
 export const parseMultiline = (value: string | undefined): string[] | undefined => {
   return value?.split(/,|\n/).map((variableString: string) => variableString.trim())
 }
+
+export const getDefinedInteger = (value: string | undefined): number | undefined => {
+  if (!value) {
+    return undefined
+  }
+
+  const number = parseFloat(value)
+  if (!Number.isInteger(number)) {
+    const error = Error(`${number} is not an integer`)
+    task.setResult(task.TaskResult.Failed, error.message)
+    throw error
```

Comment:
It'd be great to throw the name of the parameter that caused the exception, but let's keep that in mind for later.

Reply:
Let's just fix this one in this PR.

Reply:
Hum, it might be better to handle this at the `resolve-config` level, but we'll address that later, it's great to have it already 👍